### PR TITLE
Standardize several notations, part 1:  Renaming add/edit/delete-events and rendering functions

### PR DESCRIPTION
**Pull request by phorward**
This is one pull request for resolving the first issue in #7. I would like to integrate these changes stepwise.
I also started to integrate a simple replacement script that will be extended later on.
Please integrate it ASAP, so that further development can be performed, while still using the latest ViUR 3 master in a current customer project.

### CODE REVIEW

File: tools/viur-2to3.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
```
Comment (tsteinruecken) [inline tools/viur-2to3.py:1]: This should not be part of the core (there is no need for it to be deployed). Also it's probably better to extend the existing 2to3 tool: http://python3porting.com/fixers.html

Comment (phorward) [reply tools/viur-2to3.py:1]: Tools should be put into one place, for a fast access and central maintenance. If you want to use 2to3, feel free to integrate the changes there, I don't have time for this right now.

Comment (sveneberth) [reply tools/viur-2to3.py:1]: I agree with both of you.
Generally I would prefer to have this (and other helping stuff like examples, unittests and co.) in the same repo.
But the core will not be build or shipped as pip-package, it's integrated as submodule. Therefore it's nice to keep it small as possible.

Maybe it's better placed in our [viur-tools repo](https://github.com/viur-framework/viur-tools).

Comment (phorward) [reply tools/viur-2to3.py:1]: OK well, I don't want to have any more discussion on such little things here anymore, I have to get things done.
I'll gonna put the file into the viur-tools repo and I'm done with it.

**Comment by phorward** [comment]
OK the script is removed now, please continue with the reviewing process.

**Pull request merged**